microsoft-edge/progressive-web-apps-chromium/how-to/sidebar.md

```diff
@@ -0,0 +1,233 @@
+```
+
+When you define a preferred width in your app's manifest, the following happens:
+
+* When your app is opened in the sidebar, the sidebar is automatically resized to your preferred width.
+* Users can still resize the sidebar to make it larger than your preferred width, but not smaller.
+
+
+<!-- ====================================================================== -->
+## Build a sidebar-only app
+
+One of the most important benefits of building an app as a PWA is that, from one code base, your app adapts to all devices and operating systems, whatever their capabilities and screen sizes.
+
+By using the `edge_side_panel` manifest member, you can also make your app installable as a standalone app, or pinned to the sidebar in Microsoft Edge.
+
+However, if you prefer to build an app that can only be pinned to the sidebar but not installed as a standalone app, omit the `display` member from your web app manifest, or set its value to `browser`:
+
+```json
+{
+  "name": "PWAmp music player",
+  "lang": "en-US",
+  "start_url": "/",
+  "short_name": "PWAmp",
+  "description": "A skinable music player app to play your favorite mp3 files",
+  "display": "browser",
+  "icons": [
+    {
+      "src": "./favicon-256.png",
+      "sizes": "256x256"
+    }
+  ],
+  "edge_side_panel": {
+    "preferred_width": 480
+  }
+}
+```
+
+
+<!-- ====================================================================== -->
+## Detect when your app runs in the sidebar
+
+Detecting when your app runs in the sidebar can be useful to provide your users with the best possible experience when your app runs in this specific browser UI surface.
+
+You can detect when your app runs in the sidebar by using User-Agent Client Hints. To learn more about User-Agent Client Hints, see [Detecting Microsoft Edge from your website](../../web-platform/user-agent-guidance.md).
+
+To detect when your app runs in the sidebar, do one of the following:
+
+* On your web server, read the `Sec-CH-UA` HTTPS request header and look for the `Edge Side Panel` brand. For example:
+    
+    ```https
+    Sec-CH-UA: "Microsoft Edge";v="112", "Edge Side Panel";v="1", "Placeholder;Browser Brand";v="99"
+    ```
+
+* In the browser, use the `navigator.userAgentData` JavaScript API and read the value of the `brands` property. For example:
+
+    ```javascript
+    const brands = navigator.userAgentData.brands;
+    const sidebarBrandInfo = brands.find(b => b.brand === "Edge Side Panel");
+    if (sidebarBrandInfo) {
+      console.log(sidebarBrandInfo); // { brand: "Edge Side Panel", version: "1" }
+    } else {
+      console.log("App is not running in the Microsoft Edge sidebar");
+    }
+    ```
+
+You can also use the User Agent string to know whether your app runs in the sidebar or not.
+
+**Note**: we strongly recommend to use User Agent Client Hints instead. Using the User Agent string is an outdated mechanism for doing browser detection that has a long history of website compatibility issues.
```

Comment:
```suggestion
**Note**: we strongly recommend using User Agent Client Hints instead of the User Agent string. The User Agent string is an outdated mechanism for doing browser detection, and has website compatibility issues.
```
gentler yet still direct wording